internal/tasks/process-file-updates-task/tests/test-process-file-updates-replacements-idempotent.yaml

```diff
@@ -0,0 +1,121 @@
+          - name: setup-values
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+
+              mkdir -p "$(workspaces.pipeline.path)/$(context.pipelineRun.uid)/file-updates"
+              cd "$(workspaces.pipeline.path)/$(context.pipelineRun.uid)/file-updates"
+              mkdir mockidempotent
+              cd mockidempotent
+              git config --global init.defaultBranch main
+              git init .
+              git config --global user.email "test@test.com"
+              git config --global user.name "tester"
+
+              mkdir addons
+              cat > "addons/my-addon2.yaml" << EOF
+              indexImage:
+              name: test
+              EOF
+              git add addons/my-addon2.yaml
+              git commit -m "prior commit"
+    - name: run-task
+      taskRef:
+        name: process-file-updates-task
+      params:
+        - name: upstream_repo
+          value: "https://some.gitlab/test/mockidempotent"
+        - name: repo
+          value: "https://some.gitlab/test/mockidempotent"
+        - name: ref
+          value: "main"
+        - name: paths
+          value: >-
+            [{"path":"addons/my-addon2.yaml","replacements":[{"key":".indexImage",
+            "replacement":"|indexImage.*|indexImage: Jack|"}]}]
+        - name: application
+          value: "scott"
+        - name: file_updates_secret
+          value: "file-updates-secret"
+        - name: tempDir
+          value: "$(workspaces.pipeline.path)/$(context.pipelineRun.uid)/file-updates"
+        - name: internalRequestPipelineRunName
+          value: $(context.pipelineRun.name)
+      workspaces:
+        - name: pipeline
+          workspace: tests-workspace
+      runAfter:
+        - setup
+    - name: check-result
+      runAfter:
+        - run-task
+      params:
+        - name: fileUpdatesInfo
+          value: $(tasks.run-task.results.fileUpdatesInfo)
+        - name: fileUpdatesState
+          value: $(tasks.run-task.results.fileUpdatesState)
+        - name: tempDir
+          value: "$(workspaces.pipeline.path)/$(context.pipelineRun.uid)/file-updates"
+      taskSpec:
+        params:
+          - name: fileUpdatesInfo
+            type: string
+          - name: fileUpdatesState
+            type: string
+          - name: tempDir
+            type: string
+        steps:
+          - name: check-result
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+  
+              cd "$(params.tempDir)/mockidempotent"
+              commits=$(git log --oneline | wc -l)
+              echo "Test that only 1 commits are present"
+              test "${commits}" == "1"
+      workspaces:
+        - name: pipeline
+          workspace: tests-workspace
+  finally:
```

Comment:
This isn't necessary. We don't need to clean up dirs after a pipelineRun, only CRs